### PR TITLE
init Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md and example script

### PR DESCRIPTION
**Pull request by jagilber**
init Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md
init Scripts/sfm-connect.ps1

### CODE REVIEW

File: Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+$issuerThumbprint = ''
+$admin = $false
+Add-AzServiceFabricManagedClusterClientCertificate -ResourceGroupName $resourceGroupName `
+   -Name $clusterName `
+   -CommonName $commonName `
+   -IssuerThumbprint $issuerThumbprint `
+   -Admin:$admin
+```
+
+### Using ARM template to add common name certificate to configuration
```
Comment (sedwardstx) [inline Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md:62]: Can you reorder this so ARM example is listed first , and label it as the recommended/preferred method?

File: Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+Add-AzServiceFabricManagedNodeTypeVMSecret -ResourceGroupName $resourceGroupName `
+   -ClusterName $clusterName `
+   -Name $nodeTypeName `
+   -SourceVaultId $sourceVaultId `
+   -CertificateUrl $certificateUrl `
+   -CertificateStore $certificateStore `
+   -Verbose
+```
+
+### Using ARM template to add common name certificate to nodetype configuration
```
Comment (sedwardstx) [inline Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md:143]: similar to previous comment, list ARM examples first

File: Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+## Troubleshooting
+
+- Verify the common name certificate is time valid and not revoked.
+
+- Verify there are not multiple certificates installed with same common name.
+
+- Verify the common name certificate being used is installed on machine where PowerShell commands are being executed. Opening Certmgr.msc on machine will open the 'CurrentUser' certificate store.
+
+- Verify the Issuer certificate thumbprint matches 'issuerThumbprint' in cluster 'clients' configuration for common name certificate. Renewed certificates may not have the same Issuer thumbprint.
```
Comment (sedwardstx) [inline Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md:228]: we should ask them to add all possible issuer TP's in cases where the CA may have multiple possible intermediaries issuing renewals.  Digicert does this

**Review by sedwardstx — CHANGES_REQUESTED** [review]

File: Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md (hunk 1 above)
Comment (jagilber) [reply Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md]: updated

File: Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md (hunk 2 above)
Comment (jagilber) [reply Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md]: updated

File: Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md (hunk 3 above)
Comment (jagilber) [reply Security/How to Configure a Service Fabric Managed Cluster with Common Name Certificate.md]: updated

**Review by sedwardstx — APPROVED** [review]

**Pull request merged**